web/spec/supabase.yml

```diff
@@ -964,6 +967,49 @@ pages:
             { phone_confirm: true }
           )
           ```
+  invoke():
+    title: 'invoke()'
+    description: |
+      Invokes a Supabase Function. See the [guide](/docs/guides/api) for details on writing Functions.
+    $ref: '@supabase/functions-js.FunctionsClient.invoke'
+    notes: |
+      - Requires an Authorization header.
+      - Invoke params generally match the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) spec. 
+    examples:
+      - name: Basic invocation.
+        isSpotlight: true
+        js: |
+          ```js
+          const { data: user, error } = await supabase.functions.invoke('hello', {
+            body: { foo: 'bar' }
+          })
+          ```
+      - name: Specifying response type.
+        description: |
+          By default, `invoke()` will parse the response as JSON. You can parse the response in the following formats: `json`, `blob`, `text`, and `arrayBuffer`.
+        isSpotlight: true
+        js: |
+          ```js
+          const { data: user, error } = await supabase.functions.invoke('hello', {
+            responseType: 'text',
+            body: { foo: 'bar' }
+          })
+          ```
+      - name: Parsing custom headers.
+        description: |
+          Any `headers` will be passed through to the function. A common pattern is to pass a logged-in user's JWT token as an Authorization header.
+        isSpotlight: true
+        js: |
+          ```js
+          const { data: user, error } = await supabase.functions.invoke('hello', {
+            headers: { 
+              Authorization: 'Bearer ${supabase.auth.session()?.access_token}' 
```

Comment:
TODO: is this is the correct way to pass the access token?

Reply:
Yup that looks right Copple. If you are using supabase-js, this header is automatically set when a user logs in [here](https://github.com/supabase/supabase-js/blob/0c4f39c8a31da5bb71cad746aba6ef8c38722e8d/src/SupabaseClient.ts#L107)

This can also be set via the `setAuth` method functions-js exposes, so that you don't need to pass in this header every time you call invoke